vg/draw/canvas.go

```diff
@@ -269,26 +269,26 @@ func New(c vg.CanvasSizer) Canvas {
 // Supported formats are:
 //
 //  eps, jpg|jpeg, pdf, png, svg, and tif|tiff.
-func NewFormattedCanvas(w, h vg.Length, format string) (vg.CanvasWriterTo, error) {
+func NewFormattedCanvas(w, h vg.Length, color color.Color, format string) (vg.CanvasWriterTo, error) {
```

Comment:
s/color/background/g

I'm not wildly happy about additional params here, but I don't see a good alternative.

/cc @sbinet

Reply:
for this one... wider changes might be required to regularize canvas modification options across `vg` backends...

perhaps with a new `type` in the `vg` package that `vg` backend can recognize?
```go
package vg

type Option struct {
    Background color.Color
    DPI int
    // others...
}
```

well... this may warrant some more thinking...

Reply:
I think there's a tradeoff between the convenience of offering multiple customize-able ways to create canvases and the overhead of maintaining them. I'm not sure what the optimal place in the tradeoff space is, but it might make sense for this convenience function to stay as it is, and people needing customization can use another method to create the canvas.

Reply:
@ctessum So what is your position here? Stay as it is prior to this PR or as it is now in the PR?

Reply:
I've filed an alternate PR in #486.